samples/sample_hevc_fei/include/hevc_fei_encode.h

```diff
@@ -29,9 +29,11 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 class FEI_Encode
 {
 public:
-    FEI_Encode(MFXVideoSession* session, mfxHDL hdl, MfxVideoParamsWrapper& encode_pars,
-        const mfxExtFeiHevcEncFrameCtrl& def_ctrl, const msdk_char* dst_output,
-        const msdk_char* mvpInFile, PredictorsRepaking* rpck);
+    FEI_Encode(MFXVideoSession* session, mfxHDL hdl,
+        MfxVideoParamsWrapper& encode_pars, const mfxExtFeiHevcEncFrameCtrl& encodeCtrl,
+        const msdk_char* strDstFile, const msdk_char* mvpInFile,
```

Comment:
I think we need refactoring here for input/output files, but it's out of scope for this commit.